airbyte-integrations/connector-templates/source-python/integration_tests/standard_source_test.py.hbs

```diff
@@ -30,22 +30,42 @@ from base_python_test import StandardSourceTestIface
 
 
 class Source{{properCase name}}StandardTest(StandardSourceTestIface):
-    def __init__(self):
-        pass
+    """
+    If you don't need to reimplement any of the python methods below, you can consider deriving from DefaultStandardSourceTest instead
+    of StandardSourceTestIface and fill up the different .json files accordingly:
+
+        class Source{{properCase name}}StandardTest(DefaultStandardSourceTest):
+            pass
+    """
 
-    # fixme - return a ConnectorSpecification. the default implementation should suffice as long as the spec.json is in source_{{snakeCase name}}.
     def get_spec(self) -> ConnectorSpecification:
+        """
+        Return a ConnectorSpecification. the default implementation should suffice as long as the spec.json is in source_{{snakeCase name}}.
+        """
+        # load spec.json (from the source_test directory?)
         raw_spec = pkgutil.get_data(self.__class__.__module__.split(".")[0], "spec.json")
         return ConnectorSpecification.parse_obj(json.loads(raw_spec))
 
-    # fixme - return a config object for the connector. if it contains secrets, then place that object in a file called config.json in the secrets directory. if this is the case, then the default implementation below does not need to be changed.
-    # e.g. return an object that like this {"username": "airbyte", "password": "my_password" } that matches the schema described in the spec.json.
     def get_config(self) -> object:
-        raw_spec = pkgutil.get_data(self.__class__.__module__.split(".")[0], "config.json")
+        """
+        Return a config object for the connector.
+        if it contains secrets, then place that object in a file called config.json in the secrets directory.
+        if this is the case, then the default implementation below does not need to be changed.
+            e.g. return an object that like this {"username": "airbyte", "password": "my_password" } that matches the
+            schema described in the spec.json.
+        """
+        # load config.json (from the secrets directory?)
```

Comment:
is there an open question here?

Reply:
Depends if you have secrets or you can locate it somewhere else if you really want it seems...

But with @sherifnada 's PR all of this are changing if I understand right

Reply:
I think this can potentially go on in the `gradle.build` file in my PR instead of this file; the use is still expected to provide these files, just not from python code.